client/src/utils/quicknav/QuickNav.container.js

```diff
@@ -135,10 +136,21 @@ class QuickNavContainerWithRouter extends Component {
   }
 
   onSuggestionSelected(event, { suggestion, suggestionValue, suggestionIndex, sectionIndex, method }) {
+    if (this.bar.get("suggestions") === undefined)
+      return;
+
     const selectedSuggestion = this.bar.get("suggestions")[sectionIndex].suggestions[suggestionIndex];
     this.bar.set("selectedSuggestion", selectedSuggestion);
   }
 
+  onSuggestionHighlighted({ suggestion }) {
+    if (suggestion === null)
```

Comment:
Similar to above, I have a preference for `== null` which also will cover undefined.

```suggestion
    if (suggestion == null)
```